docs/src/app/components/pages/components/FloatingActionButton/ExampleComplex.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import FloatingActionButton from 'material-ui/lib/floating-action-button';
+import ToggleStar from 'material-ui/lib/svg-icons/toggle/star';
+import Colors from 'material-ui/lib/styles/colors';
+
+const FloatingButtonExampleComplex = () => (
+  <div>
+    <FloatingActionButton
+      iconClassName="muidocs-icon-action-grade"
+      secondary={true} />
+    <FloatingActionButton mini={true} secondary={true}>
+      <ToggleStar />
+    </FloatingActionButton>
+    <FloatingActionButton
+      iconClassName="muidocs-icon-action-grade"
+      disabled={true}
+      disabledColor={Colors.grey300} />
```

Comment:
@oliviertassinari Without it, the button becomes white, or without a color. Should I add a default background color in getDefaultProps?